ambari-logsearch/ambari-logsearch-config-solr/src/main/java/org/apache/ambari/logsearch/config/solr/LogLevelFilterManagerSolr.java

```diff
@@ -130,7 +130,7 @@ public LogLevelFilterMap getLogLevelFilters(String clusterName) {
         }
       }
     } catch (Exception e) {
-      LOG.error("Error during getting log level filters: {}", e);
+      LOG.error("Error during getting log level filters: {}", e.getMessage());
```

Comment:
Is it ok if the stacktrace is lost?

Reply:
its expected errors will be generated on startup, probably in an another PR i can add something like LOG.isDebug() condition to log the whole stacktrace